packages/components/Checkbox/src/Checkbox.types.ts

```diff
@@ -19,14 +16,24 @@ export interface ICheckboxState extends IPressableState {
    * Whether the Checkbox is disabled or not
    */
   disabled?: boolean;
+}
 
+/**
+ * Values that can be set via props or via tokens
+ */
+export interface CheckboxTokenProps {
   /**
-   * Determines position of Checkbox. True if boxSide='end'
+   * Allows you to set the checkbox to be at the before (start) or after (end) the label
    */
-  boxAtEnd?: boolean;
+  boxSide?: 'start' | 'end';
```

Comment:
Out of curiosity, why make this a token instead of a prop?

Reply:
I actually made it both a token and a prop. This allows someone to change the default to boxSide: 'end' if they want.

Reply:
Think of it as a token, that happens to be overridable via props. The value below, `tokensThatAreAlsoProps` determines which values get overridden from props.